glide.yaml

```diff
@@ -1,8 +1,7 @@
 package: go.uber.org/fx
 import:
-- package: github.com/uber-go/zap
-  # Pinned temporarily while zap reaches 1.0 (https://github.com/uber-go/fx/pull/243)
-  version: c064b5c44b285a7e2fd5c9b26e8c38228ce2bccb
+- package: go.uber.org/zap
+  version: v1.0.0-rc.1
```

Comment:
Any breaking change in the RC will also break your interface, so I'm not sure what this buys you.

I've had this discussion about interfaces with Peter and Josh before, and it's nuanced enough that we should take it out of PR comments. I'll send you a short proposal, and we can discuss in person. I'm certainly not blocking you from shipping what you want in FX, but I don't think that this interface will be helpful.